ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/FooCommandGenerator.scala

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Can we retain the old name of this file for this PR? So that we preserver the GIT history.
We can fix the file name in a later PR

Reply:
I've did the rename in a separate commit so git history is preserved.